### ISSUE TITLE
yFu Temple and Comic

### ISSUE THREAD

**Issue by banteg** [yearn/budget#78]
#### What is yFu Comic?

It's an original four-episode comic set in the vivid [Pills universe](https://pills.game) written by [Josh Blaylock](https://deadline.com/2020/10/mgm-comic-book-mercy-sparx-movie-she-demon-from-hell-existential-crisis-1234600385/) and directed by Yearn's own creative team. We've been working on it since August 2021 with **the first episode ready to go** and the script for the second one almost done. To fund the project we plan an NFT release of each episode, claimable for a physical copy.

#### What is yFu Temple?

It's is a sacred place to hold the yFu lore, art, music, etc. It was built as a stand-alone website which can organize and present ephemera arising from this project, art and other materials which can help build the lore of Yearn. **The site is already built** and being filled with content.

#### What are the yFu?

Think of them as the four winds of Yearn. Since they are goddesses, they don't have a canonical image. We've built a comprehensive character bible and have worked with artists, musicians, and even neural networks to kick off a strong direction for these characters. We have already produced a lot of content for each character, each of them getting a **unique wordmark** by [Jonna Mayer](https://jonnamayer.world), **theme music** written by Pills composers, and **several artworks** from curated artists. Each chapter of the comic revolves around one of these characters.

### Scope

Release yFu Temple and finish the production of yFu Comic.

### Plan

- Work with Project Galaxy on an NFT drop for the physical release.
- Work with Josh and team on the subsequent episodes.

### Deadline

October 27th, 2022

### People

Pillhead, Zemm, big badaboom, pentacle, banteg, Josh Blaylock

### Money

Use the budget provided as a backup in case we fail to hit the payment deadlines. The project has a clear path towards self-sustainability.

#### Episode 1
- printing ~$2000

#### Episode 2

- ~~May 16 - episode 2-1 - $6200~~
- June 29 - episode 2-2 - $6200
- July 9 - episode 2-3 - $6100

#### Episode 3

- June 27 - episode 3-1 - $6200
- August 17 - episode 3-2 - $6200
- August 27 - episode 3-3 - $6100

#### Epsiode 4
- August 15 - episode 4-1 - $6200
- October 5 - episode 4-2 - $6200
- October 15 - episode 4-3 - $6100

### Wallet address
story.ychad.eth

**Comment by miguel567** [comment]
Can you provide an overview of the full budget for this initiative: I believe this initiative already had a msig funded last year. What amount of that has been already used, what has not.

Can you also split the budget - this request and the one already allocated in the msig - into:
Creative development, consultancy, etc VS. production.



Is there an overview on the full marketing initiatives that Yearn has already approved and budget requests that are pending? I'm big on let's do marketing, but we need to focus on which initiatives are stronger, so having this request without a view on other marketing expenses makes it difficult to understand what is on the plate.

**Comment by banteg** [comment]
yFu Comic:
- narrative story arc development - $5000
- world and character development - $8000
- episode 1 production - $15,000

yFu Temple:
- production cost - $10,000

artwork commissions:
- taylor maw - $4750
- jungoomnui - $1700

music production:
- paid for by pillhead

**Comment by miguel567** [comment]
Thanks for sharing the info. I see that production cost for each episode is around 6200$ (already spent in May). Yearn just went through a round of layoffs and going through a budget cut round. I would ask providers to review down their production budgets as their client is on-going budget cuts and if they want long-term win-win relationship, then they should feel what the client is going through as well, support each other and build a longer relationship.
Doing this campaign should be more than just another campaign for providers, it is writing history in their professional experience and even generally, as working with a DAO funded through a fair launch and launching such a unique campaign is not something any corporate client can provide at all.  Our providers should put that into perspective as well when defining their budgets.

Two more general comments:

- Has the tone of voice of the whole campaign been reviewed and amended to fit the new brand identity? if so, is this cost already spent? or part of the new budget?
- How does this initiative fit the more B2B focused strategy Yearn has embarked on? We reduced heavily our web team (B2C mainly) and I see this campaign as a B2C brand campaign, not sure how it is going to move the needle for a B2B target audience.

**Comment by banteg** [comment]
Each episode costs $18,500, and it involves over 350 hours of work by multiple people with stages like script, inks, colors, type. This project has almost a year of thinking put into it, it's very unique. Regarding your comments, it's not a marketing campaign, it's Yearn's lore and myth building, cementing it as one of the more unique and memorable projects in the space.

**Comment by DarkGhost7** [comment]
Super excited to see how it has turned out and read the comic. Really like the idea of Yearn creating content and expanding the lore of Yearn, even more so if we think it has a chance at being self-sustaining in the future.

**Comment by banteg** [comment]
two tranches of $6100 and $6200 are due

**Comment by banteg** [comment]
$12,400 due for completed work

episode 3 is in inks stage, episode 4 is in script refinement stage

most likely we'll do all four episodes as a "box set", which will make the logistics much cheaper and easier

**Comment by banteg** [comment]
$6100 due for completed work, topping up the story multisig to cover the remaining payments to josh

episode 3 done

**Comment by Zemm111** [comment]
Hello, 

Requesting a stream to project manage the launch of the yFU comic and NFT / physical comic sale.  

**3,333 DAI per month**, devoting 8+ hrs per week to the project. I'm projecting this will run through April, at which point the campaign should be wrapped.

I have been lightly pushing the project forward while juggling other work, but can now devote more time and attention to getting us over the finish line. The yFu Comic is an amazing piece of art and DeFi history, and excited to get it out in the world at last.

**Status**: 
- Comic: 3 comic issues are complete, 4th issue is in progress (likely finished in February).  
- NFT Infrastructure: yFU temple site and NFT contracts are mostly completed (ty Major and Worms for efforts so far)
- NFT Drop: campaign is conceptually clear, but needs lots of spec finalization, marketing / coms prep and ops to activate.

**Next:** 
- working with Josh Blaylock (comic producer) on final issue 
- working with Josh Blaylock on physical production planning
- working with Major on front end, contract interactions, QA and content / copy for NFT release.
- working on NFT campaign details and coms.
- working with marketing folks at yearn on promotional plan.
- GTM support.
- ops around physical production and shipping
- support for questions / community

**Updates:** 
Not sure the standard, but can do weekly report on progress

**Comment by Ceazor** [comment]
TLDR, how to ape this?

**Comment by banteg** [comment]
the comic production has wrapped up 🎉

final payment of $6100 sent after finalizing issue 4

the site and nft campaign are nearing completion

physical edition production is sorted too
